src/index.ts

```diff
@@ -56,8 +56,8 @@ function loadConfig (): configType {
   // build config object
   const config: configType = rc('mps')
 
-  if (!config.web_admin_password || !config.web_admin_user || !config.jwt_secret) {
-    logger.error('Web admin username, password and jwt secret are mandatory. Make sure to set values for these variables.')
+  if ((config.web_auth_enabled && (!config.web_admin_password || !config.web_admin_user)) || !config.jwt_secret) {
```

Comment:
I think to increase readability -- we can break this up a bit. To me, JWT and Web Auth are two separate pieces and shouldn't be checked together.
``` suggestion
if(config.web_auth_enabled){
    if(!config.web_admin_password || !config.web_admin_user){
//log error and exit
     }
}

if(!config.jwt_secret){
//log error and exit
}
```

Reply:
done